tests/operator/operator.go

```diff
@@ -888,11 +893,11 @@ spec:
 			Eventually(fetchVirtHandlerCommand, 60*time.Second, 5*time.Second).Should(ContainSubstring(maxDevicesCommandArgument))
 
 			By("Deleting patch from KubeVirt object")
-			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
+			newVMIPerNodeRemovePatch, err := patch.New(
+				patch.WithRemove("/spec/configuration/virtualMachineInstancesPerNode")).GeneratePayload()
 			Expect(err).ToNot(HaveOccurred())
 
-			kv.Spec.Configuration.VirtualMachineInstancesPerNode = nil
-			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(context.Background(), kv, metav1.UpdateOptions{})
+			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Patch(context.Background(), kv.Name, types.JSONPatchType, newVMIPerNodeRemovePatch, metav1.PatchOptions{})
```

Comment:
I think this isn't needed, as it's being done automatically either way:
https://github.com/kubevirt/kubevirt/blob/070129537bc2a567a19b6a1db89ae31959128b78/tests/tests_suite_test.go#L163-L170

Reply:
correct, but below that we're checking that the virt-handler command does not contain the `--max-devices` arg any longer, and i don't think the cleanup function is called in between. Please correct me if I'm wrong.

Reply:
Yeah you're right :)
thanks!